glossary.qmd

```diff
@@ -0,0 +1,353 @@
+
+#### Multithreading
+
+A manner of scaling computing, to allow more operations to happen at the same time.
+
+Think of a glass of water. Synchronous computing is akin to having one straw: when you've finished drinking all you wish to drink, you give the straw to your friend for them to drink. [Asynchronous](#asynchronous) computing is akin to sharing the straw between you and your friend. There's still only one straw, but you can hand off sips. Parallel computing (including multithreading and multiprocessing) is like having two straws, where both you and your friend can drink out of the glass at the same time.
+
+#### Numpy
+
+The foundational Python library for managing n-dimensional array data.
+
+#### Object storage (Cloud storage)
+
+Object storage, or cloud storage, refers to massively scalable [cloud](#cloud) storage solutions like [Amazon S3](#amazon-s3-s3). It is relatively cheap, able to hold files small or large, and supports reading data via [HTTP range requests](#http-range-request). Most open geospatial data is hosted in such cloud storage solutions.
+
+#### OGR
+
+A widely-used open-source library for converting between different [vector data](#vector-data) formats, as well as reprojecting between [coordinate reference systems](#coordinate-reference-system-crs).
+
+Its common command-line tools include `ogrinfo` and `ogr2ogr`. It can be used from Python with the `pyogrio` or `fiona` libraries or from R with the `sf` library.
+
+OGR is installed as part of [GDAL](#gdal).
+
+#### Overviews
+
+Downsampled (aggregated) data intended for visualization and stored as part of a file format. Overviews are part of the [COG](#cloud-optimized-geotiff-cog) specification, and allow reading "zoomed out" data without needing to read and downsample from "full resolution" data.
+
+#### Parquet
+
+A file format for tabular data with [internal chunking](#chunk) and [internal compression](#internal-compression).
+
+#### PDAL
+
+The _Point Data Abstraction Library_, a widely-used open-source library for converting between different [point cloud data](#point-cloud-data) formats and managing point cloud data.
+
+#### PMTiles
+
+A type of [cloud-optimized](#cloud-optimized) [archive format](#archive-format) for tiled data. It can be used with either [vector](#vector-data) or [raster](#raster-data) tiled data, but is most often used with [Mapbox Vector Tile](#mapbox-vector-tile) files. The individual tiles in a PMTiles file are accessible via [HTTP range requests](#http-range-request).
+
+#### Point Cloud Data
+
+A type of geospatial data storing three dimensional point locations along with attributes for each point. Point cloud data may come from LIDAR sensors or other photogrammetry, and may represent three-dimensional terrain or buildings.
+
+#### PROJJSON
+
+A [projection definition](#coordinate-reference-system-crs) that uses JSON for encoding.
```

Comment:
term encoding might be confusing in this context, perhaps `formatting` or `style`. An example of when this might be used?

Reply:
Projjson is linked to from the `epsg` definition, and will probably be referenced from the geoparquet doc. I don't think the glossary needs an example, beyond describing that it's a way to store a CRS and linking to CRS